### DISCUSSION TITLE
Updates to AEO-light 2

### DISCUSSION THREAD

**Discussion by wilsbacher** [usc-imi/aeo-light#47] [Announcements]
We are planning a code refresh over the next year that will rework some of back end to improve stability and make modest changes to the UI to address user feedback. If you have suggestions please share them. We cannot promise we will be able to implement this time around, but we'd like to hear from users.

**Comment by ochoypico** [comment]
Hello,

Thank you very much for your work

Wish list:

-VU-meters
-Dolby filters

Regards



*José Luis Sanz*


*www.ochoypico.com <http://www.ochoypico.com>(+34) 629 56 02 38*

**Comment by ziggyphenocious** [comment]
It would be very helpful to have a way to meter audio levels when
setting image processing and other settings. I can get an idea by
looking at the waveform amplitude, but it would be very helpful to have
a dBfs meter. 

There is an intermittent bug when checking New Sample if you are in
96khz Sampling Rate. Sometimes it will sound "underwater". If you change
to 48khz, it's fine. Other times, 96khz sampling rate will sound normal.
I cannot find any reason why. I've found that 48khz Sampling rate always
works fine. When in 96khz sample rate, the output result will always be
okay, but the monitoring of New Sample will sometimes sound
"underwater", or like a sampling rate mismatch. 

I've noticed that on some long reels with numerous sound start blips,
that sometimes there will be an accumulating error as the reel
progresses. Sync will be fine at the start, but can be 6 frames out
after 100 feet or so, and then 20 frames after 300 feet. 

We are scanning the Hearst Newsreel collection on a Scanity film
scanner. It does fine for positive scans, but it is not good at doing
negative variable density optical sound. So, I have been using AEO for
negative variable density audio. 

Craig Nichols 

The Packard Humanities Institute 

On 2022-03-08 06:06, Greg Wilsbacher wrote: 

> We are planning a code refresh over the next year that will rework some of back end to improve stability and make modest changes to the UI to address user feedback. If you have suggestions please share them. We cannot promise we will be able to implement this time around, but we'd like to hear from users. 
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe [2].
> You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

 

Links:
------
[1] https://github.com/usc-imi/aeo-light/discussions/47
[2]

**Comment by terminalbeach** [comment]
Hi. 

Fantastic project. I recently saw an announcement about AEO-light being updated with the ability to extract metadata such as timecode. I am attempting to find the announcement source or additional details. Thanks.

**Comment by retokromer** [comment]
From the “issues”:
- [CineForm integration](https://github.com/usc-imi/aeo-light/issues/26)

**Comment by av-geeks** [comment]
A wonderful project. I've got some scans that are a little off - need to be rotated ever so slightly so that the soundtrack aligns from frame to frame. It would be nice to have a feature to do this in AEO-Light instead of having to rotating the file in Resolve or After Effects and hoping I did it enough.

**Comment by fabriziocarraro** [comment]
When using a video as a source (on mac) aeo-light asks if the video has to be buffered, (this in some cases increases the speed performances). This is ok for single jobs, but it is a problem when adding many jobs in the queue, because the queue stops at each job asking if I want to buffer the next video. Is there a way to set as a default that videos have to be buffered (or not buffered)? It does not make sense to have a queue that stops between jobs, isn't it?

Furthermore: the "Frame pitch start", "Frame pitch end" and "Search size" when setting overlaps would be nicer if shown in pixels instead of a float from 0 to 0.35 and 0 to 1.

Thanks

Fabrizio Carraro

**Reply by wilsbacher** [reply]
Hi Fabrizio, we are planning to default buffer all video input in the next update. We'll also look into the pros/cons of converting from float to actual pixel.
